### PR TITLE
Add models ide helper script

### PR DESCRIPTION
**Pull request by weeman1337**
I've frickeled the [ide model helper](https://github.com/barryvdh/laravel-ide-helper) into a script. Since it's not 100 % laravel I write a separate file and not into the models directly. It's quite convenient during development.

### CODE REVIEW

File: bin/gen_ide_helper_models.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env php
```
Comment (MyIgel) [inline bin/gen_ide_helper_models.php]: It would be nicer if the file has no `.php` extension ;)

File: bin/gen_ide_helper_models.php [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+require_once __DIR__ . '/../includes/application.php';
+
+/**
+ * This script generates the models helper file "_ide_helper_models.php" into the project root.
+ */
+
+$config = $app->get('config');
+$config['ide-helper.model_locations'] = ['src/Models'];
+
+$fileSystem = new \Illuminate\Filesystem\Filesystem();
```
Comment (MyIgel) [inline bin/gen_ide_helper_models.php]: I'm  a big friend of the `use` statement

File: composer.json [hunk 3]
```diff
@@ -49,7 +49,8 @@
         "filp/whoops": "^2.3",
         "phpunit/phpunit": "^8.1",
         "squizlabs/php_codesniffer": "^3.5",
-        "symfony/var-dumper": "^4.3"
+        "symfony/var-dumper": "^4.3",
+        "barryvdh/laravel-ide-helper": "^2.6"
```
Comment (MyIgel) [inline composer.json]: `b` should be somewhere at the start of the list

File: bin/gen_ide_helper_models.php [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+$command = new \Barryvdh\LaravelIdeHelper\Console\ModelsCommand($fileSystem);
+$command->setLaravel($app);
+$command->run(
+    new \Symfony\Component\Console\Input\ArrayInput([
+        '--nowrite' => true,
```
Comment (MyIgel) [inline bin/gen_ide_helper_models.php]: Is there a reason why it should not be written directly?

Comment (weeman1337) [reply bin/gen_ide_helper_models.php]: Yes like you found out in the other PR. This means writing to the model files. But then also stuff like the id property will be written. The idea here is: use it for generation to a file where you can grab the parts you need from.

File: bin/gen_ide_helper_models.php (hunk 1 above)
Comment (weeman1337) [reply bin/gen_ide_helper_models.php]: ✓ done

File: composer.json (hunk 3 above)
Comment (weeman1337) [reply composer.json]: ✓ done

File: bin/gen_ide_helper_models.php (hunk 4 above)
Comment (MyIgel) [reply bin/gen_ide_helper_models.php]: Well, that makes sense :D

**Comment by MyIgel** [comment]
This commit is merge ready but I would wait with merging until https://github.com/barryvdh/laravel-ide-helper/commit/25b61c56fed54cdec2459045bbae04ade5c8fe2c is in a `barryvdh/laravel-ide-helper` release (should be `v2.6.6`)

**Comment by cbluoss** [comment]
v2.6.6 of barryvdh/laravel-ide-helper was released last week.

**Pull request closed without merging**